docs/workforce-management-framework/topics/con-performance-guidelines.adoc

```diff
@@ -1,9 +1,9 @@
 [id='{context}-performance-guidelines']
 = Performance Guidelines
 
-{PROJECT_NAME} integrates multiple technologies and frameworks into single solution. Developers can build on top of the provided applications and use technologies that aren't part of the standard applications.
+{PROJECT_NAME} integrates multiple technologies and frameworks into single solution. Developers can build on top of the provided applications and use technologies that are not part of the standard applications. Node.js based server can rely on customer preffered database or external API. This may have significant impact on overal application performance.
 
-Node.js based server can rely on customer preffered database or external API. This  may have significant impact on overal application performance. {PROJECT_NAME} was desigined to expose all configuration into single files.
+{PROJECT_NAME} was desigined to expose all configuration into single files.
```

Comment:
Spelling ```designed ```
{PROJECT_NAME} was designed to expose all configuration into single files.